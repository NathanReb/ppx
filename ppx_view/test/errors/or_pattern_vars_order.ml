let f x =
  match%view x with
  | Pair (x, y)
  | Record {fst = y; snd = x} -> x - y
