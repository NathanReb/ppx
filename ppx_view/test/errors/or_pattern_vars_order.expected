File "or_pattern_vars_order.ml", line 3, characters 4-45:
Error: ppx_view: Branches of an or-pattern should define the same variables in the same order
